web/main/templates/casebook_page.html

```diff
@@ -27,6 +27,35 @@
             </div>
         {% endif %}
         <div class="casebook-inner">
+
+            {% if 'published' in request.GET%}
+                <div class="published-notice">
+                    <div class="top-strip-notice"></div>
+
+                    <div class="published-text">
+                        <div class="congratulations">Congratulations!</div>
+
+                        Your casebook is live! Use this link to share your book with
+                        your students  
+                        
+                        <div class="link">
+                            <a href="{{ abs_url }}">{{ abs_url }}</a>
+                            <button onclick="navigator.clipboard.writeText('{{ abs_url}}')">
+                            Click me to copy
+                            </button>
+                        </div>
+
+                        We also encourage you to share your work with your
+                        colleagues or on social media! Your book is an important
+                        contribution to open education for law and we hope you'll
+                        help others find it and use it in their own classrooms.
+                        Because you are a verified professor, your book will
```

Comment:
This text says "because you're a verified professor", so the entirety of the message needs to be behind a check for whether that's true.